internal/deviations/deviations.go

```diff
@@ -625,3 +625,10 @@ func ISISLspMetadataLeafsUnsupported(dut *ondatra.DUTDevice) bool {
 func QOSQueueRequiresID(dut *ondatra.DUTDevice) bool {
 	return lookupDUTDeviations(dut).GetQosQueueRequiresId()
 }
+
+// GetEcnSameMinMaxThresholdUnsupported returns true for devices that don't support Minimum and Maximum threshold values are same
+// EcnMaxMimumThreshold returns the maximum threshold value while comparing for pass or fail conditions.
+// CISCO: Maximum and mimium threshold value not be same. Anything less Maximum threshold should be 6144 greate then mimnimum threshold value.
```

Comment:
Lots of typos here, please revise.

s/mimimu/minimum
s/less Maximum/less than maximum
s/greate then mimnimum/greater than minimum

Reply:
revised